src/storage/postgres/src/main/java/org/locationtech/geogig/storage/postgresql/PGStorage.java

```diff
@@ -470,10 +470,10 @@ private static void createIgnoreDuplicatesRule(Connection cx, String tableName)
     private static void createObjectTableIndex(Connection cx, String tableName)
             throws SQLException {
 
-        String index = String.format("CREATE INDEX %s_objectid_h1_hash ON %s USING HASH(((id).h1))",
+        String index = String.format("CREATE INDEX %s_objectid_h1 ON %s )",
```

Comment:
I think this should be ;
String index = String.format("CREATE INDEX %s_objectid_h1 ON %s (((id).h1))",

see
https://github.com/locationtech/geogig/blob/master/src/storage/postgres/geogig_postgres.sql#L126